ext/mbstring/libmbfl/mbfl/mbfl_convert.c

```diff
@@ -30,6 +30,7 @@
 
 #include <stddef.h>
 
+#include "mbstring.h"
```

Comment:
I'd rather not add a reverse dep from libmfl to mbstring. That makes the layering cyclic. We should return additional information rather than touching globals.

Reply:
Thanks for the good point.

For what it's worth, I don't think the separation between mbstring and libmbfl is helping anything. It will be better if libmbfl is flattened into mbstring.

But even if and when that happens, these conversion functions might be useful to call from somewhere else, and if we poke into global variables then that won't be possible, so... will fix.

Reply:
This is the one place still remaining where I poke at those global variables. Will fix this, probably tomorrow.